canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -396,7 +396,7 @@ export default class PropertiesController {
 					controlValue = PropertyUtils.convertObjectStructureToArray(control.valueDef.isList, control.subControls, controlValue);
 				}
 
-				this.updatePropertyValue(propertyId, controlValue, true);
+				this.updatePropertyValue(propertyId, controlValue, true, "initial_load");
```

Comment:
`_addToControlValues()` method is called during init only. Everywhere you're calling `this.updatePropertyValue()` from this method, you should pass `"initial_load"`.